src/main/java/com/endava/soa_5/basic/Exercise_4.java

```diff
@@ -0,0 +1,40 @@
+package com.endava.soa_5.basic;
+
+public class Exercise_4 {
+    //Write 3 methods that calculates sum of numbers from 0 to a given number
+
+    int number;
+
+    Exercise_4(){}
+
+    Exercise_4(int number){
+        this.number=number;
+    }
+
+    int sumForLoopResult() {
+        int result=0;
+        for (; number > 0; ) {
+            result += number;
+            number--;
```

Comment:
move the number-- in the for declaration. This way, your code will be 1 line shorter

Reply:
Done!